static/js/stream_settings_ui.js

```diff
@@ -492,6 +510,8 @@ export function redraw_left_panel(left_panel_params = get_left_panel_params()) {
         );
     }
     maybe_reset_right_panel();
+    const is_streams_panel_empty = hidden_ids.size === stream_ids.length;
+    render_empty_left_panel_message(is_streams_panel_empty);
```

Comment:
Can we just not pass any parameter to this function and instead compute whether the left panel is empty in the function itself.